server/pulp/plugins/util/misc.py

```diff
@@ -142,3 +143,89 @@ def clear_directory(path, skip_list=()):
 
         elif os.path.isfile(entry_path):
             os.unlink(entry_path)
+
+
+def common_path(dirs):
+    _common_path = os.path.commonprefix(dirs)
+    if not _common_path.endswith("/"):
+        _common_path = os.path.dirname(_common_path)
+    return _common_path
+
+
+def make_link_unit(unit, extra_dst_path, working_dir,
+                   remote_repo_path, remote_root, configuration, extra_src_path, out_path):
+    """
+    This method creates symlink in working directory, pointing to remote
+    content directory, so they can be sync to remote server
+
+    :param repo_type: type of repository
+    :type repo_type: str
+    :param unit: Pulp content unit
+    :type unit: Unit
+    :param extra_dst_path: ending path for processes units
```

Comment:
I think I know what this is getting at, but it should probably be re-worded.for clarity.